front/app/components/UI/FormComponents/index.tsx

```diff
@@ -218,14 +222,16 @@ export const FormLabel = memo<
       ])}
     >
       <LabelContainer>
-        {propsHasValues(props) ? (
-          props.labelValue
-        ) : (
-          <FormattedMessage
-            {...props.labelMessage}
-            values={props.labelMessageValues}
-          />
-        )}
+        <StyledSpan>
```

Comment:
@brentguf I just wanted to check - is FormattedMessage considerd inline or block-level? I just wanted to double check for accessibility.

Reply:
I think it renders a span by default, but not sure. You can inspect the DOM to figure it out? :) I've never had an issue with it, so wouldn't worry about it actually. Or why would it be an a11y issue?

Reply:
@brentguf Ok I'll take a look. Thank you! In our notion article on a11y it says "To ensure the HTML is valid, Inline elements (span, a, button, etc.) should not contain block-level elements."